src/configLoaders/ReactConfigLoader.ts

```diff
@@ -0,0 +1,35 @@
+import ConfigLoader from './ConfigLoader';
+import createReactJestConfig from '../utils/createReactJestConfig';
+import * as path from 'path';
+
+export default class ReactConfigLoader implements ConfigLoader {
+  private loader: NodeRequire;
+  private projectRoot: string;
+
+  public constructor(projectRoot: string, loader?: NodeRequire) {
+    this.loader = loader || /* istanbul ignore next */ require;
+    this.projectRoot = projectRoot;
+  }
+
+  public loadConfig(): string {
```

Comment:
Some comments in this method are not useful to me. For example the comment above `return JSON.stringify(jestConfiguration);`.